test/ui/test_access_store_ui.py

```diff
@@ -0,0 +1,25 @@
+from exasol.nb_connector.ui.access_store_ui import get_access_store_ui
+from IPython.display import display
+
+
+def test_access_store_ui_screenshot(solara_test, page_session, assert_solara_snapshot):
+    display(get_access_store_ui())
+    box_element = page_session.locator(":text('Configuration Store')").locator('..').locator('..')
+    box_element.wait_for()
+    assert_solara_snapshot(box_element.screenshot())
+
+def test_enter_password_and_click_open(solara_test, page_session,assert_solara_snapshot):
+    display(get_access_store_ui())
+    password_input = page_session.locator("input[type='password']")
+    password_input.wait_for()
+    password_input.fill("dummy123")
+
+    open_button = page_session.locator("button:text('Open')")
+    open_button.wait_for()
+    open_button.click()
+
+    box_element = page_session.locator(":text('Configuration Store')").locator('..').locator('..')
+    box_element.wait_for()
+    box_element.screenshot()
+
+    assert_solara_snapshot(box_element.screenshot())
```

Comment:
add also the loading of the secret store using https://github.com/exasol/notebook-connector/blob/main/exasol%2Fnb_connector%2Fsecret_store.py#L23